src/Module/Api/Friendica/Index.php

```diff
@@ -23,6 +23,7 @@
 
 use Friendica\DI;
 use Friendica\Module\BaseApi;
+require_once __DIR__ . '/../../../../include/api.php';
```

Comment:
why do we need this here?

Reply:
Otherwise the `api_call` does fail. Previously this had been included in the `BaseAPI`. But since this module here is just for the transition phase it fits here better.